recipe/meta.yaml

```
@@ -34,11 +34,12 @@ requirements:
     - numpy
     - swig
     - libitk-devel
+    - libitk
```

```suggestion
    - libitk =5.1
```
Let's just see if it's libitk related?